src/pages/test-report/cat-d/components/test-report.cat-d.components.module.ts

```diff
@@ -4,13 +4,13 @@ import { IonicModule } from 'ionic-angular';
 import { DirectivesModule } from '../../../../directives/directives.module';
 import { ComponentsModule } from '../../../../components/common/common-components.module';
 import { TestReportComponentsModule } from '../../components/test-report-components.module';
-import { VehicleChecksCompactCatDComponent } from './vehicle-checks-compact/vehicle-checks-compact.cat-d';
+import { VehicleChecksComponent } from './vehicle-checks/vehicle-checks';
 import { SafetyQuestionsCatDComponent } from './safety-questions/safety-questions.cat-d';
 import { PcvDoorExerciseComponent } from './pcv-door-exercise/pcv-door-exercise';
 
 @NgModule({
   declarations: [
-    VehicleChecksCompactCatDComponent,
+    VehicleChecksComponent,
     SafetyQuestionsCatDComponent,
```

Comment:
The `SafetyQuestionsCatDComponent` is not used anymore so would be a good idea to remove it from the declarations. And actually, the entire component can be deleted

Reply:
Ok I didn't know about this, thanks

Reply:
@szabikr the component has been removed